CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 This file documents all notable changes in `LogDNA LogSpout Image`. The release numbering uses [semantic versioning](http://semver.org).
 
+## v1.3.0 - Release on April 20, 20201
```

Comment:
@darinspivey, sorry for the nit but is it `Release` or `Released` just to make sure it aligns with others too?

Reply:
@darinspivey, + `20201` is a typo, I guess